config/istio-install/generate.sh

```diff
@@ -12,9 +12,7 @@ if [[ ${istioctl_version} != "${DESIRED_ISTIO_VERSION}" ]]; then
 fi
 
 echo "generating Istio resource definitions..." >&2
-mkdir -p "${SCRIPT_DIR}/../../istio"
```

Comment:
I think this was responsible for the empty `istio` dirs we've been seeing show up in `cf-k8s-networking` locally.